### DISCUSSION TITLE
How to Create an Inline Header?

### DISCUSSION THREAD

**Discussion by zicklag** [typst/typst#3277] [Q&A] [answered]
Hey there! Does anybody know how to create an inline header like in this screenshot?

![image](https://github.com/typst/typst/assets/25393315/538aaf90-fa0b-4758-b63c-d7a2c3b56ab1)

I can't figure out how to get the header to flow inline with the next paragraph.

**Comment by andreasKroepelin** [comment, marked as answer]
```typ
#show heading.where(level: 5): it => strong(it.body) + [ ]

===== This is inline with
the following text.
```

![SmartSelect_20240127_023304_Discord.jpg](https://github.com/typst/typst/assets/42342396/99d27d29-1085-468e-9c9f-634d6e5acefd)

**Reply by zicklag** [reply]
Ah, I get it, that avoids the paragraph break. Thanks!
